.github/ISSUE_TEMPLATE/Bug_issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Report a bug
+about: Report an issue that you have encountered.
+title: ''
+labels: 'Type: bug'
+---
+
+#### Summary
+
+<Provide information about the problem.>
+
+#### Steps to reproduce
+
+<Provide CIL version and instructions for reproducing this issue.>
+
+#### Expected behaviour
+
+<Explain how the software should behave.>
+
+#### Current behaviour
+
+<Provide information about the current behaviour.>
```

Comment:
would swap these two

Reply:
I deleted them and substituted with Edo's  sentence posted above